starlette/templating.py

```diff
@@ -27,14 +27,14 @@ def __init__(
         self,
         template: typing.Any,
         context: dict,
+        content: typing.Optional[typing.Any] = None,
```

Comment:
```suggestion
        content: typing.Optional[str] = None,
```

Reply:
The `content` attribute in the parent `Response` is of type `Any`. I'm not sure it's necessarily of type `str` here.

Reply:
Well in this case it is always 'str' as output of 'render' sync and async in Jinja2.
I might be missing something but we could narrow the type down here as 'str' and that would still satisfy the parent with 'Any'.
What do you think @Kludex?

Reply:
I'm not even sure if the `Optional` is needed. :sweat_smile: 

But yeah, it should be `str`. :+1: 

The return type of the render methods is `str`, and `_TemplateResponse` is a private class.